Example/EpoxyExample/ViewControllers/LayoutGroups/DynamicLayoutGroupsViewController.swift

```diff
@@ -0,0 +1,71 @@
+// Created by Tyler Hedrick on 11/11/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+import Epoxy
+import UIKit
+
+final class DynamicLayoutGroupsViewController: CollectionViewController {
+
+  // MARK: Lifecycle
+
+  init() {
+    super.init(layout: UICollectionViewCompositionalLayout.list)
+    setItems(items, animated: false)
+  }
+
+  // MARK: Private
+
+  private enum DataID: CaseIterable {
+    case row1
+    case row2
+    case row3
+  }
+
+  private var openOptions: [AnyHashable: Bool] = [:]
+
+  private var items: [ItemModeling] {
+    var result = [ItemModeling]()
+    for id in DataID.allCases {
+      result.append(
+        DynamicRow.itemModel(
```

Comment:
nit maybe the following instead of collecting items into the result array? 
```swift
private var items: [ItemModeling] {
  DataID.allCases.map { id in
    …
  }
}
```